nextflow.config

```diff
@@ -23,6 +23,7 @@ params {
     clustering_resolutions       = '0.5,1.0'
     celltypist_model             = ''
     cellbender_epochs            = 150
+    expected_cells               = 0
```

Comment:
`expected_cells` is not a pipeline parameter, but a per-sample parameter in the samplesheet. So this is not needed.
```suggestion
```